src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -127,7 +138,7 @@ const ValueElement = (props) => {
     >
       <List.Item key={bucket.key}>
         <Checkbox
-          label={label}
+          label={label ? label : bucket.key}
```

Comment:
Is this check necessary since the label is defined above and it will always have a value?

Reply:
👍